pulsar-metadata/src/main/java/org/apache/pulsar/metadata/api/MetadataStoreConfig.java

```diff
@@ -29,6 +29,9 @@
 @Getter
 @ToString
 public class MetadataStoreConfig {
+    public static final String LOCAL_METADATA_STORE = "local-metadata-store";
+    public static final String STATE_METADATA_STORE = "state-metadata-store";
```

Comment:
Interesting, I don't know we have a zookeeper-based function state store

I think we don't need to have a separate name for this one? because it will use the same zookeeper with the metadata store right?

And I would suggest changing "local-metadata-store" -> "metadata-store", just to keep consistent with the configuration name that we have in the broker.conf.

Reply:
>And I would suggest changing "local-metadata-store" -> "metadata-store", just to keep consistent with the configuration name that we have in the `broker.conf`.
I agree with that. Today the user knows this from configuration: `metadataStoreUrl` and `configurationMetadataStoreUrl` so when they see a label name, they need to be able to match it with what they configured, no?